src/main/webapp/i18n/en/textExercise.json

```diff
@@ -14,6 +14,8 @@
             },
             "wordCount": "Number of words: {{ count }}",
             "characterCount": "Number of characters: {{ count }}",
+            "create": "Create Text",
+            "import": "Import Text",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys